### PR TITLE
Update periodic boundary circuit cutting tutorial to the new format

### PR DESCRIPTION
**Pull request by henryzou50**
## Summary

Reorganizes the tutorial around a small simulator example followed by a consolidated hardware workflow, consistent with the updated tutorial template.

- Adds learning outcomes, prerequisites, and next steps.
- Modernizes Qiskit APIs and selects a chain from backend connectivity.
- Corrects periodic ZZ averaging and explains sampling overhead.
- Uses least-busy backend selection with a 120-qubit minimum.
- Refreshes plots and saved hardware results, with a rounded two-minute usage estimate.

**Validation:** Executed the simulator example and a 33-qubit hardware test on `ibm_boston`, using 102 seconds of QPU time. Python lint, notebook validation, Markdown, spelling, and internal-link checks passed.

### CODE REVIEW

File: docs/tutorials/periodic-boundary-conditions-with-circuit-cutting.ipynb [hunk 1]
```diff
@@ -1,1005 +1,633 @@
...
+      "metadata": {},
+      "source": [
+        "---\n",
+        "title: Circuit cutting for periodic boundary conditions\n",
+        "description: Use gate cutting to implement periodic boundary conditions, then compare reconstructed observables with an uncut circuit.\n",
+        "---\n",
+        "\n",
+        "# Circuit cutting for periodic boundary conditions\n",
+        "\n",
+        "*Usage estimate: 2 minutes on ibm_boston (NOTE: This is an estimate only. Your runtime may vary.)*"
```
Comment (abbycross) [inline docs/tutorials/periodic-boundary-conditions-with-circuit-cutting.ipynb]:
```suggestion
        "*Usage estimate: 2 minutes on `ibm_boston` (NOTE: This is an estimate only. Your runtime may vary.)*"
```

File: docs/tutorials/periodic-boundary-conditions-with-circuit-cutting.ipynb [hunk 2]
```diff
@@ -1,1005 +1,633 @@
...
-     "data": {
-      "text/plain": [
-       "<Image src=\"/docs/images/tutorials/periodic-boundary-conditions-with-circuit-cutting/extracted-outputs/79428537-66cf-40ce-87cf-0f75f591cb4b-0.avif\" alt=\"Output of the previous code cell\" />"
+      "cell_type": "markdown",
+      "id": "periodic-01",
+      "metadata": {},
+      "source": [
+        "## Learning outcomes\n",
+        "\n",
+        "After completing this tutorial, you will be able to:\n",
```
Comment (abbycross) [inline docs/tutorials/periodic-boundary-conditions-with-circuit-cutting.ipynb:25]: We can remove this preamble content on line 25

File: docs/tutorials/periodic-boundary-conditions-with-circuit-cutting.ipynb [hunk 3]
```diff
@@ -1,1005 +1,633 @@
...
-       "<Image src=\"/docs/images/tutorials/periodic-boundary-conditions-with-circuit-cutting/extracted-outputs/ad38aa32-4613-46c5-bf62-da332a1b9dfb-0.avif\" alt=\"Output of the previous code cell\" />"
+      "cell_type": "markdown",
+      "id": "periodic-02",
+      "metadata": {},
+      "source": [
+        "## Requirements\n",
+        "\n",
+        "Before starting this tutorial, install:\n",
+        "\n",
+        "- Qiskit SDK v2.3 or later, with [visualization](/docs/api/qiskit/visualization) support (`pip install 'qiskit[visualization]~=2.3.1'`)\n",
```
Comment (abbycross) [inline docs/tutorials/periodic-boundary-conditions-with-circuit-cutting.ipynb]:
```suggestion
        "- Qiskit SDK v2.3 or later, with [visualization](/docs/api/qiskit/visualization) support\n",
```
(We don't specify because not every machine can use that specific syntax to install qiskit visualization.)

File: docs/tutorials/periodic-boundary-conditions-with-circuit-cutting.ipynb [hunk 4]
```diff
@@ -1,1005 +1,633 @@
...
+        "- Qiskit Runtime v0.46 or later (`pip install 'qiskit-ibm-runtime~=0.46.1'`)\n",
+        "- Circuit cutting Qiskit addon v0.10.0 (`pip install 'qiskit-addon-cutting~=0.10.0'`)\n",
+        "- Qiskit Aer v0.17.2 or later (`pip install 'qiskit-aer~=0.17.2'`)\n",
+        "\n",
+        "The hardware example also requires an [IBM Quantum Platform account](https://quantum.cloud.ibm.com/) with access to a QPU with at least 120 qubits and [saved credentials](/docs/guides/save-credentials). The simulator example runs without an account."
```
Comment (abbycross) [inline docs/tutorials/periodic-boundary-conditions-with-circuit-cutting.ipynb]:
```suggestion
        "The hardware example also requires an [IBM Quantum&reg; Platform account](https://quantum.cloud.ibm.com/) with access to a QPU with at least 120 qubits and [saved credentials](/docs/guides/save-credentials). The simulator example runs without an account."
```

File: docs/tutorials/periodic-boundary-conditions-with-circuit-cutting.ipynb [hunk 5]
```diff
@@ -1,1005 +1,633 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "periodic-18",
+      "metadata": {},
+      "source": [
+        "## Large-scale hardware example\n",
+        "\n",
+        "Reuse the workflow on the least busy operational QPU with at least 120 qubits. Select a longest shortest path in its undirected coupling graph: each consecutive pair is connected, while the endpoints are far apart. This adapts the chain to the selected device instead of relying on a fixed Eagle layout. The code checks that the chain contains at least 30 qubits and that its endpoints are not neighbors.\n",
```
Comment (abbycross) [inline docs/tutorials/periodic-boundary-conditions-with-circuit-cutting.ipynb]:
```suggestion
        "Reuse the workflow on the least-busy operational QPU with at least 120 qubits. Select a longest shortest path in its undirected coupling graph: each consecutive pair is connected, while the endpoints are far apart. This adapts the chain to the selected device instead of relying on a fixed Eagle layout. The code checks that the chain contains at least 30 qubits and that its endpoints are not neighbors.\n",
```

File: docs/tutorials/periodic-boundary-conditions-with-circuit-cutting.ipynb [hunk 6]
```diff
@@ -1,1005 +1,633 @@
...
+      "id": "periodic-20",
+      "metadata": {},
+      "source": [
+        "In the recorded run on `ibm_boston`, the cut estimates were approximately $M_Z=-0.923$ and $C_{ZZ}=0.896$, compared with $-0.611$ and $0.516$ for the uncut circuit. Both cut estimates were closer to the ideal values of $-1$ and $+1$.\n",
+        "\n",
+        "Compare each result with its ideal value and inspect the circuit depths and total QPU usage. Cutting removes the need to route the periodic gates, but introduces intermediate operations and additional statistical uncertainty. Hardware noise and calibration affect the comparison; an improvement is not guaranteed. Repeated runs are needed to assess whether an observed difference is statistically significant.\n",
+        "\n",
+        "## Next steps\n",
+        "\n",
+        "You have used a virtual connection to compile a periodic circuit, cut its boundary gates, and reconstructed observables from Sampler results.\n",
```
Comment (abbycross) [inline docs/tutorials/periodic-boundary-conditions-with-circuit-cutting.ipynb:584]: This sentence is unexpected for a "Next steps" section. Necessary? Could move it to be the last line of the previous section, if it feels important to preserve.

File: docs/tutorials/periodic-boundary-conditions-with-circuit-cutting.ipynb [hunk 7]
```diff
@@ -1,1005 +1,633 @@
...
+      "id": "periodic-21",
+      "metadata": {},
+      "source": [
+        "## Tutorial survey\n",
+        "\n",
+        "Please take this short survey to provide feedback on this tutorial. Your insights will help us improve our content offerings and user experience.\n",
+        "\n",
+        "[Link to survey](https://your.feedback.ibm.com/jfe/form/SV_3fQQYAIjTxvIChg)\n",
+        "\n",
+        "© IBM Corp. 2025, 2026"
```
Comment (abbycross) [inline docs/tutorials/periodic-boundary-conditions-with-circuit-cutting.ipynb:606]: Are we starting to include the tutorial survey section/link again intentionally? Last I heard, we were no longer using it (@nathanearnestnoble might know for sure?) and we can also remove the copyright statement on line 606.

File: docs/tutorials/periodic-boundary-conditions-with-circuit-cutting.ipynb (hunk 2 above)
Comment (henryzou50) [reply docs/tutorials/periodic-boundary-conditions-with-circuit-cutting.ipynb]: Done!

File: docs/tutorials/periodic-boundary-conditions-with-circuit-cutting.ipynb (hunk 6 above)
Comment (henryzou50) [reply docs/tutorials/periodic-boundary-conditions-with-circuit-cutting.ipynb]: Agreed, it wasn't adding much. I removed it so "Next steps" now leads straight into the Recommendations admonition, matching the other circuit cutting tutorials.

File: docs/tutorials/periodic-boundary-conditions-with-circuit-cutting.ipynb (hunk 7 above)
Comment (henryzou50) [reply docs/tutorials/periodic-boundary-conditions-with-circuit-cutting.ipynb]: Not intentional. I missed removing it when updating the tutorial. I've dropped the whole survey cell, including the copyright line.
